conf/declaration.routes

```diff
@@ -1,8 +1,8 @@
 # Declaration type
 
-GET         /dispatch-location                   controllers.declaration.DispatchLocationController.displayPage(mode: models.Mode ?= models.Mode.Normal)
+GET         /dispatch-location                   controllers.declaration.DispatchLocationController.displayPage()
 
-POST        /dispatch-location                   controllers.declaration.DispatchLocationController.submitForm(mode: models.Mode ?= models.Mode.Normal)
+POST        /dispatch-location                   controllers.declaration.DispatchLocationController.submitForm()
```

Comment:
I prefer being forced to think about which Mode the page needs to be in, it is easy to create bugs without this

Reply:
Fixed - now mode backs to constroller signature